tasktiger/worker.py

```diff
@@ -131,7 +132,7 @@ def _update_queue_set(self, timeout=None):
             r, w, x = select.select([fileno], [], [],
                                     0 if self._queue_set else timeout)
             if fileno in r: # or not self._queue_set:
-                message = gen.next()
+                message = six.next(gen)
```

Comment:
We can just use `next(gen)`, right? I don't care about support for Python older than 2.7.